README.md

```diff
@@ -4,7 +4,8 @@ Shows the rendered HTML of the AsciiDoc content in the current editor.
 
 You can install this module from the command-line by typing `apm install asciidoc-preview`.
 You can toggle the preview pane, which appears to the right of the editor, using the key combination `ctrl-shift-a`.
+More options are accessed in by right-clicking on the preview.
 
 Thanks to @kevinsawicki! I just adapted [markdown-preview](https://github.com/atom/markdown-preview) and used [asciidoctorjs-npm-wrapper](https://github.com/anthonny/asciidoctorjs-npm-wrapper).
 
-![](https://raw.githubusercontent.com/anthonny/asciidoc-preview/master/screenshot.jpg)
+![AsciiDoc Preview demo](https://cloud.githubusercontent.com/assets/6715247/11607696/0f555d54-9b49-11e5-8c72-31b718289562.gif)
```

Comment:
Sure. The previous version linked to an absolute URL in a different (the original?) repo. The docs suggest that a relative link should work though.